plugins/dql-backend/src/service/queries.ts

```diff
@@ -87,4 +88,54 @@ export const dynatraceQueries: Record<
     | fieldsRemove id, name, workload.labels, cluster.id, namespace.id
     | fieldsAdd Environment = "${apiConfig.environmentName}"`;
   },
+  [DynatraceQueryKeys.SRG_VALIDATIONS]: (entity, apiConfig) => {
+    const catalogTags =
+      entity.metadata.annotations?.['dynatrace.com/guardian-tags'];
+    let filterString = '';
+
+    if (catalogTags) {
+      const tags = catalogTags.split(',').map(tag => {
+        const [key, value] = tag.split('=');
+        if (value === undefined) {
+          return { [key]: undefined };
+        }
+        return { [key]: value || '' };
+      });
+
+      if (tags.length > 0) {
+        filterString += '| filter ';
+        filterString += tags
+          .map(tag => {
+            const key = Object.keys(tag)[0];
+            const value = tag[key];
+            return value !== undefined
+              ? `in (tags[${key}], "${value}")`
```

Comment:
To be on the safe side, we can escape the key, so that spaces in keys are allowed
```suggestion
              ? `in (tags[\`${key}\`], "${value}")`
```